auth0/src/main/java/com/auth0/android/authentication/AuthenticationAPIClient.java

```diff
@@ -592,7 +592,8 @@ public SignUpRequest signUp(@NonNull String email, @NonNull String password, @No
     }
 
     /**
-     * Requests new Credentials using a valid Refresh Token.
+     * Requests new Credentials using a valid Refresh Token. How the new Credentials are requested depends on the {@link Auth0#isOIDCConformant()} flag.
```

Comment:
Let's explain this a bit better, at least talk about the two endpoints involved.